layers/+completion/spacemacs-helm/config.el

```diff
@@ -0,0 +1,14 @@
+;;; config.el --- Spacemacs-helm Layer config File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar spacemacs-helm-basic-config nil
+  "If non nil, don't bind keys globally in spacemacs-helm and
+don't enable helm-mode.")
```

Comment:
`spacemacs-helm-basic` can be rename to something better like `spacemacs-helm-base` maybe ? We already have `spacemacs-base` layer.